client/extensions/woocommerce/components/process-orders-widget/index.js

```diff
@@ -15,12 +15,15 @@ import { getLink } from 'woocommerce/lib/nav-utils';
 const ProcessOrdersWidget = ( { className, site, orders, currency, ordersRevenue, translate } ) => {
 	const classes = classNames( 'card', 'process-orders-widget__container', className );
 	const currencyValue = currency && currency.value || '';
+	const orderCountPhrase = translate( 'New order', 'New orders', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **24** times:
`translate( 'Save New Order' )` *ES Score: 7*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).